instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
@mtyaka Oh, good, that makes things a whole lot easier.  In that case, I would add `EDXAPPS_SETTINGS` now; it saves us the trobule of thinking of it later, and the only downside I can think of is that it might confuse someone, which can be addressed by a simple comment.

Regarding the file storage class, I was wrongly assuming that the code in https://github.com/open-craft/edx-platform/blob/omar/remove-s3/lms/envs/openstack.py#L22-L27 handles this for us, but since we explicitly set this in Ansible, we have to override it there as well.